.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't think IDE stuff should get committed as it's developer-specific. My py3.7 is elsewhere, for instance.

Reply:
and added .vscode to the .gitignore file